style.css

```diff
@@ -54,18 +54,52 @@ body {
 }
 
 .menu__item {
-  font: 500 18px/21px 'Roboto', serif;
-	padding: 0 13px;
+  position: relative;
+  font: 500 18px/21px 'Roboto', sans-serif;
+  margin-right: 26px;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5;
+}
+
+.menu__item_active::after {
+  position: absolute;
+  display: block;
+  content: '';
+  width: 100%;
+  height: 3px;
+  left: 0;
+  background: #0056ec;
 }
 
 .menu__item:last-child {
-	padding: 0;
+  margin-right: 0;
+}
+
+.menu__item:hover {
+  opacity: .5;
+}
+
+.menu__item:hover::after {
+  position: absolute;
+  left: 0;
+  display: block;
+  content: '';
+  width: 100%;
+  height: 3px;
+  background: #0056ec;
+}
+
+.menu__item:hover:after,
+ .menu__item_active::after {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/second_final_np/pullRequest?prid=6232598)